src/setup/extract-configuration.go

```diff
@@ -29,20 +29,21 @@ import (
 	"stellar/util"
 )
 
-//Configuration is the schema for all experiment configurations.
+// Configuration is the schema for all experiment configurations.
 type Configuration struct {
 	Sequential     bool            `json:"Sequential"`
 	Provider       string          `json:"Provider"`
+	Runtime        string          `json:"Runtime"`
```

Comment:
This will allow us to set the provider "default" runtime. However, serverless.com allows to overwrite default runtime for each functione. Consider adding an optional runtime field for each subexperiment so that we can deploy defferent runtime functions in a single experiment. (This is not implemented in yet, but could be usefull in the future. Maybe create a separate PR if you do not think it is relevant in this one. It's your call :D  )
See serverless documentation
https://www.serverless.com/framework/docs/providers/aws/guide/functions


```
functions:
  hello:
    handler: handler.hello # required, handler set in AWS Lambda
    runtime: python3.11 # optional overwrite, default is provider runtime
```